draft-ietf-dnsop-dns-capture-format-00.md

```diff
@@ -1490,74 +1515,42 @@ expected from that starting point; the stronger compression produces
 files that are 150% of the size of similarly compressed binary format,
 and require over 4x more CPU to compress.
 
-## Binary serialisation formats
-
-### Implementation notes
-
-* CBOR.  CBOR is a simple format, and simple to implement. The CBOR website
-  lists implementations for 16 languages.
-* Protocol Buffers. Code must be generated for a particular data schema to
-  to read and write data using that schema. The Google code generator can
-  currently generate code for encoding and decoding a schema for C++, Go,
-  Java, Python, Ruby, C#, Objective-C, Javascript and PHP.
-* Avro. Avro libraries are available for C, C++, C#, Java, Python, Ruby
-  and PHP. Optionally tools are available for C++, Java and C# to generate
-  code for encoding and decoding.
-
-Avro and Protocol Buffers both allow storage of untagged data, but
-because they rely on the data schema for this, their implementation is
-considerably more complex than CBOR. Using Avro or Protocol Buffers in
-an unsupported environment would require notably greater development
-effort compared to CBOR.
-
-### Performance
+## Performance
 
 Concentrating again on the block formats, all three produce format
 files that are close to an order of magnitude smaller that the
-original PCAP.  Avro produces the smallest files and CBOR the
-largest. The Avro file is 80% the size of CBOR files, with the
-Protocol Buffers file 90% of the CBOR size.
+original PCAP.  CBOR produces the largest files and Avro
+the smallest, 20% smaller than CBOR.
 
 Once compression is taken into account, the size difference
-narrows. At medium compression (gzip), Avro is 96% and Protocol
-Buffers 98% of CBOR size, and using strong compression Protocol
-Buffers is the smallest and Avro the largest, with Protocol Buffers
-being 98% and CBOR 99% the size of Avro, though CBOR and Protocol
-Buffers require more compression CPU.
+narrows. At medium compression (gzip), the size difference is 4%, and
```

Comment:
At medium compression (using gzip) the size difference is 4%. Using strong compression (with gzip) the difference reduces to 2%, with Avro the largest and Protocol Buffers the smallest, although CBOR and Protocol Buffers require slightly more compression CPU.